docs/src/Nlay_tutorial.md

```diff
@@ -0,0 +1,151 @@
+# N-layer Cylinder Examples
+
+A major functionality of this package is the ability to perform forward modeling of the diffusion equation in arbitrarily layered media. This page will detail how to use the Nlayer code including the packages `fluence_DA_Nlay_cylinder_CW`, `fluence_DA_Nlay_cylinder_TD`, and `fluence_DA_Nlay_cylinder_FD`. There are also other ppackages for Nlayerd cylinders such as `flux_DA_Nlay_cylinder_CW` that can be used in a similar manner.
+
+## 2-Layer Example
+The first example we will walk through is a simple 2-layered media. We will show how to simulate individual fluence values for the steady-state, time-domain, and frequency-domain as well as how to simulate a vector of fluence values at various times or source detector separations.
+
+Below is an image of the 2-layer system we will be modeling, including the optical properties and thicknesses of each layer as well as the radius of the cylinders shown (a). Note that for most scenarios, you will want to consider a sufficiently large radius to approximate a semi-infinate lateral geometry.
+
+![2lay](./assets/2lay.png)
+
+### Steady-state example
+First we will go through the steady state example using the function `fluence_DA_Nlay_cylinder_CW`. This function has the following arguments including optional keyword arguments after the semicolon that default to a 2-layer medium:
+
+```julia
+fluence_DA_Nlay_cylinder_CW(ρ, μa, μsp; n_ext=1.0, n_med=(1.0, 1.0), l=(1.0, 5.0), a=10.0, z=0.0, MaxIter=10000, atol=eps(Float64))
+```
+
+For this example we will be ignoring the key word arguments for MaxIter and atol which deal with the precision of the calculation. For detailed information on what each of these arguments are, see the docs by typing `? fluence_DA_Nlay_cylinder_CW` in your julia window.
+
+Below are the values you would put into this function to model the two-layered media in the figure above assuming ρ = 1 cm. Notice that μa, μsp, n_med, and l all must have the same number of arguments as you want layers in the model:
+
+```julia
+julia> using LightPropagation
+julia> fluence_DA_Nlay_cylinder_CW(1.0, (0.20, 0.15), (15.0, 20.0); n_ext=1.0, n_med=(1.0, 1.0), l=(0.4, 0.8), a=20.0, z=0)
+0.007359921451026246
+```
+
+NOTE: make sure you put all arguments in decimal point form, i.e. μsp= 15.0 not 15, or there will be an error.
```

Comment:
I don't think this is true anymore however, it is recommended to ensure types stability. 
```julia
julia> fluence_DA_Nlay_cylinder_CW(1, (0.1, 0.2), (10.0, 12), n_med=(1.0, 1))
0.023755152650609134
```